near/omni-bridge/src/lib.rs

```diff
@@ -911,6 +1112,27 @@ impl Contract {
         self.destination_nonces.get(&chain_kind).unwrap_or_default()
     }
 
+    #[private]
+    pub fn resolve_transfer(&mut self, _amount: U128) -> U128 {
+        U128(0)
+    }
```

Comment:
What is the purpose of this dummy method?

Reply:
It's to avoid refunds as decided here https://github.com/Near-One/omni-bridge/pull/147#discussion_r1883275127